README.md

```diff
@@ -71,15 +69,15 @@ This is then used to power things like [contract testing](#contract-testing), [g
 
 ## Dereference
 
-Aliases include: "dereferencing", "internal inlining" or "transclusion".
+Aliases include: "Dereferencing", "Internal Inlining" or "Transclusion".
 
-All `$ref`s and replaced with their values a'la copy & paste. No more `$ref`'s exist in the file/object representation. If you have 10 operations referencing the same model 10 times, you now have 10 different models.
+All `$ref`s and replaced with their values à la copy & paste. No more `$ref`'s exist in the file/object representation. If you have 10 operations referencing the same model 10 times, you now have 10 different models.
 
 ## Description Validation
 
-Aliases include: "schema validation".
+Aliases include: "Schema Validation".
 
-A very different type of validation to [data validation](#data-validation). Description validation focuses on making sure the description is correct against the OpenAPI Specification itself.
+A very different type of validation to [data validation](#data-validation). Description validation focuses on making sure the [API Description Document](#api-description) is correct against the OpenAPI Specification itself.
```

Comment:
Again description is fine because there is a description document which uses a description format but those two parts make a description.